src/_guides/language/concurrency/index.md

```diff
@@ -462,6 +458,42 @@ is slower when isolates are in different groups.
   Flutter doesn't support `Isolate.spawnUri()`.
 {{site.alert.end}}
 
+## Concurrency on the web
+
+All Dart apps can use `async-await`, `Future`, and `Stream`.
```

Comment:
I am *slightly* worried about mixing "concurrency" and asynchronous computation, as if they are the same.
Asynchronous computations are still single-threaded (the opposite of concurrent), they just provide a practical way to yield control to other computations.

Maybe add:

> ... `Stream` for non-blocking, interleaved computations.
> Actual concurrency using isolates, however, is not ...

(Asynchronous functionality is great for *waiting for* other concurrent computations, because you don't block, but they are not themselves concurrent.)

Reply:
All the async=concurrency content in these PRs is because I misunderstood [this comment](https://github.com/dart-lang/site-www/issues/4042#issuecomment-1290683024) from the original issue, but I get it now! Close one 😅